contrib/atuin/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "atuin"
+pkgver = "18.1.0"
+pkgrel = 0
+build_wrksrc = "atuin"
+build_style = "cargo"
+hostmakedepends = ["cargo"]
+makedepends = ["sqlite-devel"]
```

Comment:
--no-default-features to remove `check-update`

Reply:
you can also replace build_wrksrc with `-p atuin` to the args while doing that, and then remove the `../` from the install below i think

Reply:
(since it also spends minutes in install, so replace that with do_install and a manual one as well)